ctp2_code/gs/fileio/civscenarios.cpp

```diff
@@ -519,12 +522,14 @@ CIV_SCEN_ERR CivScenarios::MakeNewScenario(ScenarioPack *pack, MBCHAR *scenName,
 
 	MBCHAR scenPath[_MAX_PATH];
 	sprintf(scenPath, "%s%sscen%04d", pack->m_path, FILE_SEP, pack->m_numScenarios);
+
+	int rc;
 #ifdef WIN32
-	if(!_stat(scenPath, &tmpstat)) {
+	rc = _stat(path, &tmpstat);
 #else
-	if(!stat(scenPath, &tmpstat)) {
+	rc = stat(path, &tmpstat);
```

Comment:
Getting here
```
civscenarios.cpp:530:12: error: 'path' was not declared in this scope
rc = stat(path, &tmpstat);
```
Guess it shoudl be `scenPath` instead of `path`

Reply:
Will fix.